bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/items/GroupItem.java

```diff
@@ -251,7 +255,20 @@ public void stateChanged(Item item, State oldState, State newState) {
      */
     @Override
     public void stateUpdated(Item item, State state) {
+        oldGroupState = this.state; 
         setState(function.calculate(members));
+        sendGroupStateUpdatedEvent(item.getName(), this.state);

```

Comment:
Shouldn't we check if this.oldGroupState and this.state differs and send a ChangedEvent if it differs (instead of always an update).
As long as the group state is calculated by the members it could differ (perhaps relevant to the used function, e.g. state change counter).